package.json

```
@@ -43,15 +43,18 @@
     "sass-loader": "^10.1.0",
     "style-loader": "^3.3.1",
     "ts-loader": "^8.0.12",
-    "typedoc": "^0.25.3",
-    "typescript": "^5.2.2",
+    "typedoc": "^0.26.2",
+    "typescript": "^5.5.2",
     "url-loader": "^4.1.1",
     "webpack": "^5.11.1",
     "webpack-cli": "^4.3.0",
     "webpack-merge": "^5.7.3"
   },
   "peerDependecies": {
```

```suggestion
  "peerDependencies": {
```
Yeah, maybe we can fix that 👀